src/Tizen.Security.TEEC/Tizen.Security.TEEC/Libteec.cs

```diff
@@ -596,7 +625,8 @@ internal void Open64(Guid destination, uint loginMethod, byte[] connectionData,
         /// <exception cref="NotSupportedException">The required feature is not supported.</exception>
         /// <exception cref="InvalidOperationException">The operation is invalid.</exception>
         public void Close() {
-            Interop.Libteec.CloseSession(session_imp);
+            Dispose(true);
```

Comment:
Do not call Dispose in Close()

Reply:
What about this patern?:
```
public class Stream : IDisposable {  
    IDisposable.Dispose(){  
        Close();  
    }  
    public void Close(){  
        Dispose(true);  
        GC.SuppressFinalize(this);  
    }  
}  
```

Reply:
Stream dose not provide open method

Reply:
Note that Open32, Open64 are internal.
The only way to create Session is to call context.OpenSession(...) and after
call session.Close() session is not usable any more (one can't open it again).
So making Dispose() and Close() work identical makes sense.

Reply:
Ok, I understand your point

But please, consider to refactoring
Session should be created from factory method instead of context.OpenSession.
context.OpenSession just use Factory method to meet high cohesion
it is a case of Expert pattern
https://en.m.wikipedia.org/wiki/GRASP_(object-oriented_design)#Information_expert